docs/docker/README.md

```diff
@@ -0,0 +1,57 @@
+## TODO for Dockerization
+* Add maintainer(s)
+  - Maintainer for CR Dockerfile needs to be added after pull request is merged. Ideally the maintainer should be someone from official team with appropriate Docker Hub permission to tag image as `box/clusterrunner:latest`. I can volunteer if needed but that will require collaborator permission for Docker Hub.
```

Comment:
@josephharrington Currently I can push CR images to Docker Hub only using my Hub username i.e. as [`yamaszone/clusterrunner:src`](https://hub.docker.com/r/yamaszone/clusterrunner/). The official CR image should be tagged and pushed as `boxinc/clusterrunner:latest`, `boxinc/clusterrunner:0.5.473`, `boxinc/clusterrunner:0.5.474`, etc. which only users under `boxinc` and collaborators of a repo have permissions this action. I will include an environment variable `CLUSTERRUNNER_VERSION` in the Dockerfile so that there is a Docker image for every version of CR.